tests/integration/customizations/history/test_show.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import os
+import re
+
+from awscli.testutils import aws, unittest, FileCreator
+
+
+class TestShow(unittest.TestCase):
+    def setUp(self):
+        self.files = FileCreator()
+        self.environ = os.environ.copy()
+        self.environ['AWS_CONFIG_FILE'] = self.files.create_file(
+            'config', (
+                '[default]\n'
+                'cli_history = enabled'
+            )
+        )
+        self.environ['AWS_DEFAULT_PROFILE'] = 'default'
+        self.environ['AWS_DEFAULT_REGION'] = 'us-west-2'
+        self.environ['AWS_HISTORY_PATH'] = self.files.rootdir
```

Comment:
I thought we changed this to have a `CLI` prefix (e.g `AWS_CLI_HISTORY_PATH`).  Was that feedback not incorporated?

Reply:
Hmmm. I thought I changed that too. I think the test is using the wrong value.